zstackctl/zstackctl/migrate_influxdb_data.py

```
@@ -691,7 +691,7 @@ def get_migrate_time(size):
     if len(count_list) > 0:
         max_table_count = max(count_list)
         print_green("本次迁移任务最大记录: %d " % max_table_count)
-        print_green("指定单次迁移 %d 条，预估迁移时间: %d 秒" % (size, 1 if (max_table_count / size) < 10 else (max_table_count / size)))
+        print_green("指定单次迁移 %d 条，预估迁移时间: %d 秒" % (size, 1 if (max_table_count // size) < 10 else (max_table_count // size)))
```

Ruff (0.15.2)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)